content/articles/auto-import-dns.markdown

```diff
@@ -0,0 +1,48 @@
+---
+title: Auto-importing DNS records
+excerpt: Avoid downtime when transferring or hosting your domain with us by auto-importing your DNS records
+categories:
+- DNS
+---
+
+# Auto-importing DNS records
+
+Auto-importing DNS records can help you reduce downtime on your domain transfer or when hosting the DNS for your domain with us.
+
+Instead of entering manually DNS records or importing them in BIND format (not all domain providers let you export your records in this standard format), we scan existing records and import them for you.
+
+We currently can import the following DNS record types:
+- A
+- AAAA
+- CNAME
+- MX
+- TXT
+
+<note>
+This feature is available on Personal, Professional, Business, and Reseller plans
```

Comment:
You may want to look at how we are expressing this in other articles (Like LE ones). This doesn't make me think that the other plans are Legacy ones that we want to discontinue.